model/internal/data/protogen/metrics/v1/metrics.pb.go

```diff
@@ -990,22 +1030,25 @@ func (m *IntDataPoint) GetExemplars() []IntExemplar {
 // NumberDataPoint is a single data point in a timeseries that describes the
 // time-varying value of a double metric.
 type NumberDataPoint struct {
-	// The set of labels that uniquely identify this timeseries.
-	Labels []v11.StringKeyValue `protobuf:"bytes,1,rep,name=labels,proto3" json:"labels"`
-	// start_time_unix_nano is the last time when the aggregation value was reset
-	// to "zero". For some metric types this is ignored, see data types for more
-	// details.
+	// The set of key/value pairs that uniquely identify the timeseries from
+	// where this point belongs. The list may be empty (may contain 0 elements).
+	Attributes []v11.KeyValue `protobuf:"bytes,7,rep,name=attributes,proto3" json:"attributes"`
+	// Labels is deprecated and will be removed soon.
+	// 1. Old senders and receivers that are not aware of this change will
+	// continue using the `labels` field.
+	// 2. New senders, which are aware of this change MUST send only `attributes`.
+	// 3. New receivers, which are aware of this change MUST convert this into
+	// `labels` by simply converting all int64 values into float.
 	//
-	// The aggregation value is over the time interval (start_time_unix_nano,
-	// time_unix_nano].
+	// This field will be removed in ~3 months, on July 1, 2021.
+	Labels []v11.StringKeyValue `protobuf:"bytes,1,rep,name=labels,proto3" json:"labels"` // Deprecated: Do not use.
```

Comment:
Has the Labels field been removed in the latest version? We still use the old version of the collector in some places. Is there any way to make it compatible?